system/Cache/Handlers/PredisHandler.php

```diff
@@ -28,13 +27,17 @@ class PredisHandler extends BaseHandler
      * @var array
      */
     protected $config = [
-        'scheme'   => 'tcp',
```

Comment:
Removed 'scheme' since this is not exposed via Cache.php and is not used throughout this file. I have a future to-do for myself to add TLS support, then maybe we can reintroduce scheme here as well as in Cache.php and RedisHandler.php, and set it up to accept either tls or tcp.